.travis.yml

```diff
@@ -1,16 +1,54 @@
 language: rust
-rust:
-  - nightly
-  - beta
-  - stable
-env:
-  - GTK=3.4
-  - GTK=3.18
+matrix:
+  include:
+    - os: linux
+      rust: nightly
+      env: GTK=3.4
+    - os: linux
+      rust: nightly
+      env: GTK=3.18
+    - os: linux
+      rust: beta
+      env: GTK=3.4
+    - os: linux
+      rust: beta
+      env: GTK=3.18
+    - os: linux
+      rust: stable
+      env: GTK=3.4
+    - os: linux
+      rust: stable
+      env: GTK=3.18
+    - os: osx
+      rust: nightly
+      env: GTK=3.4
+    # - os: osx
+    #   rust: nightly
+    #   env: GTK=3.18
+    - os: osx
+      rust: beta
+      env: GTK=3.4
+    # - os: osx
+    #   rust: beta
+    #   env: GTK=3.18
+    - os: osx
+      rust: stable
+      env: GTK=3.4
+    # - os: osx
+    #   rust: stable
+    #   env: GTK=3.18
+sudo: true
 addons:
   apt:
     packages:
     - libgtk-3-dev
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew unlink python; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install gtk+3 cairo atk; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/opt/X11/lib/pkgconfig; fi
```

Comment:
Separate ifs with same condition is special?

Reply:
Just more readable (from my point of view).

Reply:
IMHO multi-line commands is allowed.
Currently only found not full multi-command example in https://github.com/gtk-rs/gio/blob/master/.travis.yml#L15-L17,
but I seen real multiple commands somewhere.